exporter/clickhouselogsexporter/usage.go

```diff
@@ -32,23 +33,26 @@ var (
 		Measure:     ExporterSigNozSentLogRecords,
 		Description: "The number of logs exported to signoz",
 		Aggregation: view.Sum(),
-		TagKeys:     []tag.Key{usage.TagTenantKey},
+		TagKeys:     []tag.Key{usage.TagTenantKey, usage.TagExporterIdKey},
 	}
 	LogsSizeView = &view.View{
 		Name:        "signoz_logs_bytes",
 		Measure:     ExporterSigNozSentLogRecordsBytes,
 		Description: "The size of logs exported to signoz",
 		Aggregation: view.Sum(),
-		TagKeys:     []tag.Key{usage.TagTenantKey},
+		TagKeys:     []tag.Key{usage.TagTenantKey, usage.TagExporterIdKey},
 	}
 )
 
-func UsageExporter(metrics []*metricdata.Metric) (map[string]usage.Usage, error) {
+func UsageExporter(metrics []*metricdata.Metric, id uuid.UUID) (map[string]usage.Usage, error) {
 	data := map[string]usage.Usage{}
 	for _, metric := range metrics {
 		if strings.Contains(metric.Descriptor.Name, "signoz_logs_count") {
 			for _, v := range metric.TimeSeries {
-				tenant := v.LabelValues[0].Value
+				if v.LabelValues[0].Value != id.String() {
+					continue
+				}
```

Comment:
I'd use the names for comparison instead of index 0, 1 etc.

Reply:
I don't have the name 
![image](https://github.com/SigNoz/signoz-otel-collector/assets/26831659/6c52b4f3-dda2-4de5-97a4-3892a1f0efec)

Reply:
`metric.Descriptor.LabelKeys` is what you are looking for. It is a pair wise array with `LabelValues`